commit a96c9b511cedeaa8c04ae7d42289b71f1663735d (refs/changes/34/4234/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-06-15T19:01:24+00:00 (1 year, 4 months ago)
    
    TRITON-510 waferlock in sdc-manatee should support pgstatsmon
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/sapi_manifests/waferlock/template b/sapi_manifests/waferlock/template
index 5f06195..c404496 100644
--- a/sapi_manifests/waferlock/template
+++ b/sapi_manifests/waferlock/template
@@ -17,7 +17,9 @@
 	],
 	"domains": [
 		{{#dns_domain}}
-			"{{{moray_domain}}}"
+			"{{{moray_domain}}}",
+			{{! Don't use pgstatsmon_domain, might not be set. }}
+			"pgstatsmon.{{{datacenter_name}}}.{{{dns_domain}}}"
 		{{/dns_domain}}
 		{{^dns_domain}}
 			"moray.{{{DOMAIN_NAME}}}",
